oak_containers_stage1/src/main.rs

```diff
@@ -19,17 +19,18 @@
 mod client;
 mod image;
 
+use anyhow::Context;
 use clap::Parser;
 use client::LauncherClient;
 use image::Image;
 use std::error::Error;
 
 #[derive(Parser, Debug)]
 struct Args {
-    #[arg(long, required = true)]
+    #[arg(long, default_value_t = 2)]
     launcher_vsock_cid: u32,
 
-    #[arg(long, required = true)]
+    #[arg(long, default_value_t = 8080)]
```

Comment:
This is fine for now, but won't work if we want to run multiple launchers on a single host.